### PR TITLE
Test search relevancy on PRs.

### PR DESCRIPTION
**Pull request by dkinzer**
Required by https://github.com/tulibraries/tul_cob/pull/2144

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SOLR_PORT=8995
+SOLR_URL=http://localhost:${SOLR_PORT}/solr/blacklight
+SOLR_VERSION=8.3.0
```
Comment (bibliotechy) [inline .env:3]: We use 8.2.0 in production

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+include .env
+export
+
+up:
```
Comment (bibliotechy) [inline Makefile:4]: Don't all of these need to take place in the submodule directory?

File: .env (hunk 1 above)
Comment (bibliotechy) [inline .env:3]: We use `8.2.0` in production

**Review by bibliotechy — COMMENTED** [review]
My only concern is that the `.env` is going to get picked up somewhere we don't want / expect it picked up, like in airflow when we are running this. If those are only used in the Makefile, can they just be defined in their? I don't feel super strong about this, just wanted to raise it as a possible issue.

File: Makefile (hunk 2 above)
Comment (dkinzer) [reply Makefile:4]: Nope.  That's not necessary

File: .env (hunk 1 above)
Comment (dkinzer) [reply .env:3]: OK.  Thanks.

**Comment by dkinzer** [comment]
That's an interesting concern.  We're not using the dotenv gem here so unless I source and export the file in the airflow context I think it cannot get piked up.

Would it be OK to keep it for now and see what happens on airflow?  The airflow DAGs would fail pretty quickly if they try to do something with a non existent SOLR_URL.

**Review by bibliotechy — APPROVED** [review]

**Pull request merged**